dist_example.py

```diff
@@ -142,7 +82,7 @@ def test(args, model, device, test_loader, verbose=True):
             data, target = data.to(device), target.to(device)
             output = model(data)
             test_loss += F.nll_loss(
-                output, target, reduction="sum"
+                output, target, reduction = "sum"
```

Comment:
I'd run the code linter [Black](https://black.readthedocs.io/en/stable/) over the code:

``` shell
$ black dist_example.py adadamp/_dist.py
```